content/getting-started/server.md

```diff
@@ -95,7 +95,8 @@ sudo setcap 'cap_net_bind_service=+ep' ./urbit
 {% tab label="Windows" %}
 
 ```shell
-curl.exe -L https://urbit.org/install/windows/latest | tar xzk --strip=1 && ./urbit
+curl.exe -L https://urbit.org/install/windows/latest | tar xzk --strip=1
```

Comment:
```
PS C:\Users\thelifeandtimes> curl.exe -L https://urbit.org/install/windows/latest | tar xzk --strip=1                                                                         curl.exe : The term 'curl.exe' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path    was included, verify that the path is correct and try again.
At line:1 char:1
+ curl.exe -L https://urbit.org/install/windows/latest | tar xzk --stri ...                                                                                           + ~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (curl.exe:String) [], CommandNotFoundException                                                                              
    + FullyQualifiedErrorId : CommandNotFoundException
```
Maybe I don't have curl on this machine? Supposedly I'm on build 19044.1706

output of `$PSVersionTable` is:

```
Name                           Value
----                           -----
PSVersion                      5.1.19041.1682
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.1682
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```